SPECS/sm.spec

```
@@ -422,8 +416,68 @@ Manager and some other packages
 %{_unitdir}/fairlock@.service
 %{_libexecdir}/fairlock
 
+%post fairlock
+## On upgrade, shut down existing lock services so new ones will
+## be started. There should be no locks held during upgrade operations
+## so this is safe.
+if [ $1 -gt 1 ];
+then
+    /usr/bin/systemctl list-units fairlock@* --all --no-legend | /usr/bin/cut -d' ' -f1 | while read service;
+    do
+        /usr/bin/systemctl stop "$service"
+    done
+fi
 
 %changelog
+* Thu Mar 20 2025 Ronan Abhamon <ronan.abhamon@vates.tech> - 3.2.12-3.1
+- Rebase on 3.2.12-3
+- Sync patches with our latest 3.2.12-8.3 branch
+- Remove patches merged upstream:
+- - 0028-CA-398425-correctly-check-for-multiple-targets-in-iS.patch
+- - 0030-fix-getAllocatedSize-is-incorrect-75.patch
+- - 0047-Define-and-details-attr-on-Failure-mock.patch
+- - 0064-feat-add-HPE-Nimble-multipath-configuration.patch
+- *** Upstream changelog ***
+- * Tue Feb 11 2025 Mark Syms <mark.syms@cloud.com> - 3.2.12-3
+- - CA-405381: update mpathcount while xapi is not enabled
+- * Thu Feb 06 2025 Mark Syms <mark.syms@cloud.com> - 3.2.12-2
+- - CA-403593: don't log session refs
+- * Mon Jan 06 2025 Mark Syms <mark.syms@cloud.com> - 3.2.12-1
```

As mentioned in the previous comments, we can now start indenting the upstream changelog with spaces.

Example:
```suggestion
  * Tue Feb 11 2025 Mark Syms <mark.syms@cloud.com> - 3.2.12-3
  - CA-405381: update mpathcount while xapi is not enabled
  * Thu Feb 06 2025 Mark Syms <mark.syms@cloud.com> - 3.2.12-2
  - CA-403593: don't log session refs
  * Mon Jan 06 2025 Mark Syms <mark.syms@cloud.com> - 3.2.12-1
```